src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,60 @@
+import { Ref, useRef, ReactElement, forwardRef } from "react";
+import { useTab, useFocusRing } from "react-aria";
+import { Node } from "@react-types/shared";
+import { TabListState } from "react-stately";
+import { mergeProps } from "@react-aria/utils";
+import { st, classes } from "./tabs.st.css";
+export interface TabProps<T> {
+  /** Add a class to the content div. */
+  className?: string;
+  id?: string;
+  /** Tab item */
+  item: Node<T>;
+  state: TabListState<T>;
+  /** Add predefined data-id to ease testing or analytics. */
+  dataId?: string;
```

Comment:
Yeah, I agree, we should be more specific with comments; they need to be helpful since this will be open-source. I will try to make them a bit more clear..

Reply:
Yeah, one thing is if you extend `ComponentBase` that will give you the dataId so you can remove that. The comments are surfaced in the props table within the docs. So no comment then blank cell.  This is however an internal component to it matters less.